### DISCUSSION TITLE
heic incompatible

### DISCUSSION THREAD

**Discussion by Grindingstill** [immich-app/immich#14061] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

HEIC format from iphone - looked around but can't find a solution.  So, uploaded images to server but server can't see it.  Can't find a solution.  Thoughts?

### Platform

- [X] Server
- [x] Web
- [ ] Mobile

**Comment by danieldietzler** [comment]
Immich supports heic. What do you mean with "server can't see it"?

**Comment by Grindingstill** [comment]
Some suggest that there is a licensing issue and immich server can read jpeg but not heic, so the image is not shown

**Comment by mertalev** [comment]
Do you mean that the assets show up in the timeline, but there’s no thumbnail? Could you share the server logs (`docker logs immich_server`)? The docker-compose.yaml file would also be helpful.

**Comment by Grindingstill** [comment]
So I switched the phone to jpeg mode and then to heic mode.  The jpeg showed up on the server fine, but not the heic picture (it says "error loading image").

I don't know how to find the log files.

**Reply by alextran1502** [reply]
Are you using the default setup? If you can post your setup info, we can probably point out the issue. 

You can get the logs with `docker logs immich-server` 

AFAIK AppleProRaw on iOS18 right now is the one having issue generating thumbnail, other HEIC formats work well

**Reply by Grindingstill** [reply]
Thanks.  Which settings are you thinking of?  I remember one in particular.

**Reply by Grindingstill** [reply]
After looking some more, it turns out the yunohost.org immich install still has some heic problem that is still there.  Hoping to find a solution.

**Comment by movsar** [comment]
I can't even upload HEIC files, from web version says incompatible or something like that and Android app ignores them altogether
